webpack/components/EditableTextInput/EditableTextInput.js

```diff
@@ -64,10 +87,20 @@ const EditableTextInput = ({
   }, [editing]);
 
   const onClear = () => {
+    if (isPassword) {
+      if (hasPassword || inputValue?.length > 0) {
+        setPasswordPlaceholder(PASSWORD_MASK);
+      }
+    }
+
     setInputValue(value);
     setEditing(false);
   };
 
+  const toggleShowPassword = () => {
+    setShowPassword(!showPassword);
```

Comment:
```suggestion
    setShowPassword(prevPassword => !prevPassword);
```
When setting a state that depends on the previous state, I feel like using the function form of `setState` is safer because it guarantees you won't be operating on a stale value.